gto/exceptions.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, name) -> None:
 
 
 class ManyVersions(GTOException):
-    _message = "'{versions}' versions of artifact '{name}' found"
+    _message = "Multiple versions ('{versions}') of artifact '{name}' found on the same git commit. Multiple versions are not allowed."
```

Comment:
nit: Multiple versions are not allowed on the same Git commit. Found <versions> of an artifact <name> on <git ref>

wdyt?

Reply:
Sorry, I am not able to understand your suggestion: "Found of an artifact on"?

Would this be ok?
`Multiple versions are not allowed on the same Git commit. Violation detected for artifact '{name}', versions '{versions}'.`

Reply:
Just in case it is good, I pushed the change.

Reply:
thanks!